include/Reader/abisignature.h

```diff
@@ -57,13 +57,33 @@ class ABICallSignature : public ABISignature {
 private:
   const ReaderCallSignature &Signature; ///< The target function signature.
 
+  /// \brief Emits a call to an unmanaged function.
+  ///
+  /// This method is called by \p emitCall when emitting the call target is an
```

Comment:
the call target is an unmanaged function --> a call whose target is an unmanaged function